interface/weno/adminClass.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * weno rx admin class.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+class adminProperties {
```

Comment:
for me (figure out how to psr4 this, namespace it, organize in codebase, and autoload it)